src/orange/orangeinp/UnitProto.hh

```diff
@@ -0,0 +1,202 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/orangeinp/UnitProto.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "geocel/Types.hh"
+#include "orange/OrangeTypes.hh"
+#include "orange/transform/VariantTransform.hh"
+
+#include "ProtoInterface.hh"
+
+namespace celeritas
+{
+namespace orangeinp
+{
+class Transformed;
+
+namespace detail
+{
+struct CsgUnit;
+}  // namespace detail
+
+//---------------------------------------------------------------------------//
+/*!
+ * Construct a "unit" universe.
```

Comment:
Communicate that this *the* standard CSG universe, not some sort of special case